openmrs/src/test/resources/json/programEnrollment/bahmni-program-enrollment-response.json

```diff
@@ -14,18 +14,10 @@
   "attributes": [
     {
       "uuid": "614eb218-68aa-46fe-8b93-f023e5d6d825",
-      "display": "Registration Number: ASD",
-      "links": [{
-        "rel": "self",
-        "uri": "/openmrs/ws/rest/v1/bahmniprogramenrollment/8bed8bd4-7ec4-47d6-a8b9-c5141bf4b6a3/attribute/614eb218-68aa-46fe-8b93-f023e5d6d825"
-      }]
+      "value": "Registration Number: ASD"
```

Comment:
this file should look like the real response from bahmni, so please do not change this